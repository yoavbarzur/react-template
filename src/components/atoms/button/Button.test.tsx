// eslint-disable-next-line no-unused-vars
import React from 'react';
import { render } from '@testing-library/react'
import { Button } from './Button';

describe('button', () => {
    test('Button', () => {
        const { container } = render(<Button label="submit" />)
        expect(container).toMatchSnapshot();
    });
})
