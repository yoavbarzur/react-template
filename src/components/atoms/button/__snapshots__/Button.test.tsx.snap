// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button Button 1`] = `
<div>
  <button
    class="storybook-button storybook-button--medium storybook-button--secondary"
    type="button"
  >
    submit
  </button>
</div>
`;
